public/homepage/stylesheets/style.less

```diff
@@ -470,3 +481,330 @@ footer {
     .transform(scaleX(1) translateY(-150%));
   }
 }
+
+
+.page-heading {
+
+  padding: 60px 0;
+  margin-bottom: 20px;
+
+  h1 {
+    color: white;
+    padding: 0;
+    font-weight: normal;
+    font-size: 37px;
+    line-height: 42px;
+    margin: 0 0 15px 0;
+  }
+
+  p {
+    color: white;
+  }
+}
+
+.page-content {
+  margin: 45px 0 65px 0;
+  overflow: auto;
+}
+
+.page-heading h1 em {
+}
+
+.callout {
+  font-size: 18px;
+  margin-bottom: 60px;
+  line-height: 150%;
+
+  em {
+    font-weight: 600;
+    text-decoration: underline;
+  }
+}
+
+.feature-list {
+  overflow: auto;
+  margin-right: -60px;
+
+  .key {
+    background: #333;
+    background: rgba(0,0,0,.15);
+    // color: rgba(255,255,255, .7);
+    color: rgba(0,0,0,.5);
+    padding: 0px 6px;
+    border-radius: 2px;
+    font-weight: 600;
+  }
+
+  .feature {
+    overflow: auto;
+    margin-bottom: 40px;
+    padding-bottom: 5px;
+    width: 50%;
+    float: left;
+    min-height: 450px;
+    box-sizing: border-box;
+    padding-right: 60px;
+
+    .feature-description {
+      color: rgba(0,0,0,.65);
+      line-height: 150%;
+
+      strong {
+        color: rgba(0,0,0,1);
+      }
+    }
+
+    .feature-name {
+      margin: 0 0 12px 0;
+      line-height: 30px;
+    }
+
+    .feature-image {
+      width:100%;
+      margin: 0 0 30px 0;
+      box-shadow: 7px 7px 0px rgba(0,0,0,.15);
+    }
+  }
+}
+
+.feature.wide {
+  width: 100%;
+  border: solid 2px #DDD;
+  padding-top: 40px;
+  padding-bottom: 20px;
+  border-width: 2px 0 2px 0;
+  min-height: auto;
```

Comment:
It's an override of some other min-height property set on similar items and seems to do the trick.